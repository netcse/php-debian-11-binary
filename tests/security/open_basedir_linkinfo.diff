--
     int(%d)
     int(%d)
     
013- Warning: symlink(): open_basedir restriction in effect. File(%s/test/bad/bad.txt) is not within the allowed path(s): (.) in %s on line %d
013+ Warning: symlink(): File exists in /home/smamran09/php-src/tests/security/open_basedir_linkinfo.php on line 24
     bool(false)
     int(%d)
     bool(true)
--
