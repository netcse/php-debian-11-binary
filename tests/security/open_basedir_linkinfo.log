
---- EXPECTED OUTPUT
*** Testing open_basedir configuration [linkinfo] ***
bool(true)
bool(true)
bool(true)
bool(true)
bool(true)
int(%d)
int(%d)
int(%d)
int(%d)
int(%d)

Warning: symlink(): open_basedir restriction in effect. File(%s/test/bad/bad.txt) is not within the allowed path(s): (.) in %s on line %d
bool(false)
int(%d)
bool(true)
*** Finished testing open_basedir configuration [linkinfo] ***
---- ACTUAL OUTPUT
*** Testing open_basedir configuration [linkinfo] ***
bool(true)
bool(true)
bool(true)
bool(true)
bool(true)
int(2065)
int(2065)
int(2065)
int(2065)
int(2065)

Warning: symlink(): File exists in /home/smamran09/php-src/tests/security/open_basedir_linkinfo.php on line 24
bool(false)
int(2065)
bool(true)
*** Finished testing open_basedir configuration [linkinfo] ***
---- FAILED
