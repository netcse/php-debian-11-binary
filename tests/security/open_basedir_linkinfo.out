*** Testing open_basedir configuration [linkinfo] ***
bool(true)
bool(true)
bool(true)
bool(true)
bool(true)
int(2065)
int(2065)
int(2065)
int(2065)
int(2065)

Warning: symlink(): File exists in /home/smamran09/php-src/tests/security/open_basedir_linkinfo.php on line 24
bool(false)
int(2065)
bool(true)
*** Finished testing open_basedir configuration [linkinfo] ***